presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
+* Add BigQuery Connector Plugin.
+* Added support for memozing dead nodes. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.
+* Enforce NOT NULL constraint by checking values for NOT NULL columns.
+* NodeStatusService implementation is now expected to validate & check if a node is allowed to register with DiscoveryNodeManager based on node.location instead of node.id configuration parameter.
+
+JDBC Connector Changes
```

Comment:
Is this `JDBC connector changes` ? Is JDBC a connector ? I believe this should just be `JDBC Changes`

Reply:
JDBC connectors all share a common module and extend from common classes.  But that's an implementation detail to a Presto operator.  In the past, what happens is we take a look at all dependencies, and add separate sections for them.  E.g. one for MySQL connector, one for Postgres connector, etc.